packages/cloud-cognitive/src/components/NonLinearReading/NonLinearReading.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { ChevronUp16 } from '@carbon/icons-react';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg } from '../../settings';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--non-linear-reading`;
+const componentName = 'NonLinearReading';
+
+// Default values for props
+const defaults = {
+  theme: 'light',
+};
+
+/**
+ * Use non-linear reading when space is limited to share a
+ * brief, at-a-glance, summary of a concept that may require
+ * more explanation for some users.
+ */
+export let NonLinearReading = React.forwardRef(
+  (
+    {
+      // The component props, in alphabetical order (for consistency).
+      children,
+      className,
+      /* other props for NonLinearReading, with default values if needed */
+      definition,
+      theme = defaults.theme,
+      ...rest
+    },
+    ref
+  ) => {
+    const [isOpen, setOpen] = useState(false);
+
+    const handleToggle = () => {
+      setOpen((prevState) => !prevState);
+    };
+
+    const handleKeyDown = (event) => {
```

Comment:
It looks like this is getting called twice? Checking keyboard actions, it looks like the NLR pills opens and closes immediately. I think you can fix this with an `event.preventDefault()` but you probably don’t need [keydown](https://stackoverflow.com/questions/71915364/do-i-need-to-add-onkeydown-to-button-for-accessibility-compliance-wcag-2-0) at all here?

Seeing that issue though, it might be worth included [`aria-expanded`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) since NLR includes expanded and collapsed state. (Also okay to investigate in a later PR to keep this initial PR small).

Reply:
Just removing the `onKeyDown` and `handleKeyDown` worked great. This is a holdover when the button was originally a span. Accessibility test still fails, though. :/

I've added the `aria-expanded` as suggested, but clearly there's more work to be done hooking up all the different "aria" parts. Thanks for letting us pushing that back to a later date. I'll make a note.